generators/server/templates/src/main/resources/config/application.yml.ejs

```diff
@@ -71,18 +71,8 @@ zuul: # those values must be configured depending on the application specific ne
     <%_ if (serviceDiscoveryType === 'consul') { _%>
     ignoredServices: 'consul,<%= baseName %>' # Prevent Consul and the "<%= baseName %>" app itself from having a Zuul route automatically created for them
     <%_ } _%>
-
-# See https://github.com/Netflix/Hystrix/wiki/Configuration
-hystrix:
```

Comment:
I also thought this wasn't working any more - can you confirm?

Reply:
Rereading the latest docs for Finchley (and yes they are hard to find as google always redirect to old versions): it seems that indeed there were some changes in recent versions:

```
If Zuul uses service discovery, you need to configure these timeouts with the ribbon.ReadTimeout and ribbon.SocketTimeout Ribbon properties.

If you have configured Zuul routes by specifying URLs, you need to use zuul.host.connect-timeout-millis and zuul.host.socket-timeout-millis.
```
https://cloud.spring.io/spring-cloud-static/Finchley.RC1/single/spring-cloud.html#_zuul_timeouts

So @Tcharl we need to set the `zuul.timeouts` properties for the gateway while keeping the `hystrix.timeouts` for microservices.